Client/Ecosia/Extensions/Referrals+BuildType.swift

```diff
@@ -10,8 +10,6 @@ import Shared
 extension Referrals {
     static var isEnabledForBuild: Bool {
         switch AppConstants.BuildChannel {
-        case .release:
```

Comment:
I wonder if we need this `enum` at all anymore.

Reply:
yes - agree. I somehow felt we might revive it but it's better to make it clean in the first place

Reply:
@vauxhall FYI: I'll keep the logic for now for version 6.0.3 (which shall not have referrals) and we'll remove the enum for 6.1.0